opentelemetry-sdk/CHANGELOG.md

```diff
@@ -65,6 +65,12 @@ also modified to suppress telemetry before invoking exporters.
   - `HistogramDataPoint` no longer exposes `bounds` and `bucket_counts`, but
     instead offers `bounds()` and `bucket_counts()` methods that returns an
     iterator over the same.
+  - `Metric` no longer exposes `name`, `description`, `unit`, `data` fields, but
```

Comment:
The individual instrument and their datapoint related structs (such as `Sum`, `SumDataPoint`, etc.) also have some public fields where we can use getters.